packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -8,9 +8,9 @@ import {
     EmbeddedCheckoutMessenger,
     EmbeddedCheckoutMessengerOptions,
     FlashMessage,
+    PaymentMethod,
     Promotion,
-    RequestOptions,
-} from '@bigcommerce/checkout-sdk';
+ RequestOptions } from '@bigcommerce/checkout-sdk';
```

Comment:
Here might be a eslint warning:

```suggestion
    RequestOptions,
 } from '@bigcommerce/checkout-sdk';
```